tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from app import create_app
+import app
+
+
+@pytest.fixture(scope='module')
+def test_client():
+    flask_app = create_app()
+    with flask_app.test_client() as testing_client:
+        with flask_app.app_context():
+            yield testing_client
+
+
+@pytest.fixture()
```

Comment:
you may use `pytest.fixture(autouse=True)`, in this case the fixture will be automatically applied, there will be not need to write `get_mocker` everywhere like `def test_get_nonexisten(test_client, get_mocker):`

Reply:
alternatively, to make the things explicit you can:
1. create a fixture like `restaurants` (see the comment below)
2. create a function like
```
def mock_restaraunts(mocker, data):
   mocker.patch.object(app, 'restaurants', data)
```
3. and write the tests like:
```python

def test_get_nonexisten(test_client, restaurants, mocker):
    mock_restaraunts(mocker, restaurants)
    response = test_client.get('/api/restaurants/3000')
    assert response.status_code == 404
    assert {"description": "Restaurant with such ID doesn\'t exist", "name": "Not Found"} == response.get_json()

Reply:
fixed